.github/workflows/tests.yml

```diff
@@ -135,11 +133,12 @@ jobs:
           name: python-package-distributions
           path: dist/
       - name: Sign the dists with Sigstore
-        uses: sigstore/gh-action-sigstore-python@v2.1.1
+        uses: sigstore/gh-action-sigstore-python@v3.0.0
```

Comment:
Using this action isn't recommended (https://github.com/sigstore/gh-action-sigstore-python/blob/be60bbea7f8770c9c9673a8fabafec00df7f458f/README.md) because the SigStore version is encapsulated by the `pypa/gh-action-pypi-publish` action

The files modified during the pypi-publish action should be uploaded directly, right?